diff --git a/libs/lensfun/database.cpp b/libs/lensfun/database.cpp
index 7195860..855967f 100644
--- a/libs/lensfun/database.cpp
+++ b/libs/lensfun/database.cpp
@@ -819,7 +819,7 @@ lfError lfDatabase::Load (const char *errcontext, const char *data, size_t data_
     };
 
     /* Temporarily drop numeric format to "C" */
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
     _configthreadlocale(_ENABLE_PER_THREAD_LOCALE);
     setlocale (LC_NUMERIC, "C");
 #else
@@ -851,7 +851,7 @@ lfError lfDatabase::Load (const char *errcontext, const char *data, size_t data_
     g_markup_parse_context_free (mpc);
 
     /* Restore numeric format */
-#if defined(_MSC_VER)
+#if defined(__MINGW32__)
     _configthreadlocale(_DISABLE_PER_THREAD_LOCALE);
 #else
     uselocale(loc);
@@ -894,7 +894,7 @@ char *lfDatabase::Save () const
 lfError lfDatabase::Save (char*& xml, size_t& data_size) const
 {
     /* Temporarily drop numeric format to "C" */
-#if defined(_MSC_VER)
+#if defined(__MINGW32__)
     _configthreadlocale(_ENABLE_PER_THREAD_LOCALE);
     setlocale (LC_NUMERIC, "C");
 #else
@@ -1137,7 +1137,7 @@ lfError lfDatabase::Save (char*& xml, size_t& data_size) const
     g_string_append (output, "</lensdatabase>\n");
 
     /* Restore numeric format */
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
     _configthreadlocale(_DISABLE_PER_THREAD_LOCALE);
 #else
     uselocale(loc);
diff --git a/libs/lensfun/lens.cpp b/libs/lensfun/lens.cpp
index 91c4bc5..521659b 100644
--- a/libs/lensfun/lens.cpp
+++ b/libs/lensfun/lens.cpp
@@ -171,7 +171,7 @@ void lfLens::GuessParameters ()
     float minf = float (INT_MAX), maxf = float (INT_MIN);
     float mina = float (INT_MAX), maxa = float (INT_MIN);
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
     _configthreadlocale(_ENABLE_PER_THREAD_LOCALE);
     setlocale (LC_NUMERIC, "C");
 #else
@@ -264,7 +264,7 @@ void lfLens::GuessParameters ()
 
     if (!MaxFocal) MaxFocal = MinFocal;
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
     _configthreadlocale(_DISABLE_PER_THREAD_LOCALE);
 #else
     uselocale(loc);