diff -ur a/src/main.c b/src/main.c
--- a/src/main.c	2017-03-27 23:05:57.539025800 +0200
+++ b/src/main.c	2017-03-27 23:04:34.684377800 +0200
@@ -1247,7 +1247,7 @@
       xfree (user_wgetrc);
     }
 #ifdef SYSTEM_WGETRC
-  if (printf (_("    %s (system)\n"), SYSTEM_WGETRC) < 0)
+  if (printf (_("    %s (system)\n"), "wgetrc") < 0)
     exit (WGET_EXIT_IO_FAIL);
 #endif
 
diff -ur a/src/Makefile b/src/Makefile
--- a/src/Makefile	2017-03-27 22:07:46.982050300 +0200
+++ b/src/Makefile	2017-03-27 23:00:50.053017314 +0200
@@ -353,6 +353,9 @@
 am__v_lt_1 = 
 COMPILE = $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
 	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
+COMPILE_HID =  $(shell echo $(COMPILE) | sed -r "s/-I[^ ]+//g;s/-L[^ ]+//g;s/x86_64-w64-mingw32-//g;s/i686-w64-mingw32-//g;s/(-DLOCALEDIR=)[^ ]+(\/share\/locale)/\1\2/g;s/(-DSYSTEM_WGETRC=)[^ ]+(\/etc\/wgetrc)/\1\2/g;s/  */ /g")
+LINK_PREHID = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) $(LIBS) $(wget_LDADD)
+LINK_HID = $(shell echo $(LINK_PREHID) | sed -r "s/-I[^ ]+//g;s/-L[^ ]+//g;s/x86_64-w64-mingw32-//g;s/i686-w64-mingw32-//g;s/(-DLOCALEDIR=)[^ ]+(\/share\/locale)/\1\2/g;s/(-DSYSTEM_WGETRC=)[^ ]+(\/etc\/wgetrc)/\1\2/g;s/  */ /g")
 AM_V_CC = $(am__v_CC_$(V))
 am__v_CC_ = $(am__v_CC_$(AM_DEFAULT_VERBOSITY))
 am__v_CC_0 = @echo "  CC      " $@;
@@ -2358,10 +2361,9 @@
 	echo '' >> $@
 	echo '#include "version.h"' >> $@
 	echo 'const char *version_string = "1.19.1.18-e892";' >> $@
-	echo 'const char *compilation_string = "'$(COMPILE)'";' \
+	echo 'const char *compilation_string = "'$(COMPILE_HID)'";' \
 	    | $(ESCAPEQUOTE) >> $@
-	echo 'const char *link_string = "'$(CCLD) $(AM_CFLAGS) $(CFLAGS) \
-	$(AM_LDFLAGS) $(LDFLAGS) $(LIBS) $(wget_LDADD)'";' \
+	echo 'const char *link_string = "'$(LINK_HID)'";' \
 	    | $(ESCAPEQUOTE) >> $@
 
 css.c: $(srcdir)/css.l
diff -ur a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am	2017-03-27 22:05:08.604350900 +0200
+++ b/src/Makefile.am	2017-03-27 23:01:08.377113344 +0200
@@ -86,10 +86,9 @@
 	echo '' >> $@
 	echo '#include "version.h"' >> $@
 	echo 'const char *version_string = "@VERSION@";' >> $@
-	echo 'const char *compilation_string = "'$(COMPILE)'";' \
+	echo 'const char *compilation_string = "'$(COMPILE_HID)'";' \
 	    | $(ESCAPEQUOTE) >> $@
-	echo 'const char *link_string = "'$(CCLD) $(AM_CFLAGS) $(CFLAGS) \
-	$(AM_LDFLAGS) $(LDFLAGS) $(LIBS) $(wget_LDADD)'";' \
+	echo 'const char *link_string = "'$(LINK_HID)'";' \
 	    | $(ESCAPEQUOTE) >> $@
 
 css.c: $(srcdir)/css.l
diff -ur a/src/Makefile.in b/src/Makefile.in
--- a/src/Makefile.in	2017-03-27 22:06:57.392265800 +0200
+++ b/src/Makefile.in	2017-03-27 22:24:00.929560270 +0200
@@ -2358,7 +2358,7 @@
 	echo '' >> $@
 	echo '#include "version.h"' >> $@
 	echo 'const char *version_string = "@VERSION@";' >> $@
-	echo 'const char *compilation_string = "'$(COMPILE)'";' \
+	echo 'const char *compilation_string = "'$(COMPILE_HID)'";' \
 	    | $(ESCAPEQUOTE) >> $@
 	echo 'const char *link_string = "'$(CCLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(AM_LDFLAGS) $(LDFLAGS) $(LIBS) $(wget_LDADD)'";' \
