--- Makefile	2006-04-26 21:14:26.000000000 +0200
+++ Makefile.original	2017-03-21 17:47:35.444770563 +0100
@@ -143,7 +143,7 @@
 
 TOAST	= $(BIN)/toast
 UNTOAST	= $(BIN)/untoast
-TCAT	= $(BIN)/tcat
+#TCAT	= $(BIN)/tcat
 
 # Headers
 
@@ -279,7 +279,7 @@
 
 # Target rules
 
-all:		$(LIBGSM) $(TOAST) $(TCAT) $(UNTOAST)
+all:		$(LIBGSM)
 		@-echo $(ROOT): Done.
 
 tst:		$(TST)/lin2cod $(TST)/cod2lin $(TOAST) $(TST)/test-result
@@ -307,13 +307,6 @@
 
 # Toast, Untoast and Tcat -- the compress-like frontends to gsm.
 
-$(TOAST):	$(BIN) $(TOAST_OBJECTS) $(LIBGSM)
-		$(LD) $(LFLAGS) -o $(TOAST) $(TOAST_OBJECTS) $(LIBGSM) $(LDLIB)
-
-$(UNTOAST):	$(BIN) $(TOAST)
-		-rm $(RMFLAGS) $(UNTOAST)
-		$(LN) $(TOAST) $(UNTOAST)
-
 $(TCAT):	$(BIN) $(TOAST)
 		-rm $(RMFLAGS) $(TCAT)
 		$(LN) $(TOAST) $(TCAT)
