diff -Nur openal-soft/CMakeLists.txt openal-soft.b/CMakeLists.txt
--- openal-soft/CMakeLists.txt	2018-01-30 17:20:40.964580459 +0100
+++ openal-soft.b/CMakeLists.txt	2018-01-30 17:20:38.284512764 +0100
@@ -1228,6 +1228,11 @@
 SET(PACKAGE_VERSION "${LIB_VERSION}")
 IF(LIBTYPE STREQUAL "STATIC")
     SET(PKG_CONFIG_CFLAGS -DAL_LIBTYPE_STATIC ${PKG_CONFIG_CFLAGS})
+    FOREACH(l ${EXTRA_LIBS})
+        IF(NOT l MATCHES "^-.*")
+            SET(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${l}")
+        ENDIF()
+    ENDFOREACH()
 ENDIF()
 
 # End configuration
diff -Nur openal-soft/openal.pc.in openal-soft.b/openal.pc.in
--- openal-soft/openal.pc.in	2018-01-30 17:20:42.828515067 +0100
+++ openal-soft.b/openal.pc.in	2018-01-30 17:20:38.291179499 +0100
@@ -8,4 +8,5 @@
 Requires: @PKG_CONFIG_REQUIRES@
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -l@LIBNAME@ @PKG_CONFIG_LIBS@
+Libs.private: @PKG_CONFIG_LIBS_PRIVATE@
 Cflags: -I${includedir} -I${includedir}/AL @PKG_CONFIG_CFLAGS@
