diff -Nur snappy_git.orig/CMakeLists.txt snappy_git/CMakeLists.txt
--- snappy_git.orig/CMakeLists.txt	2017-07-26 17:16:05.338090518 +0200
+++ snappy_git/CMakeLists.txt	2017-07-26 17:17:35.457280186 +0200
@@ -72,9 +72,7 @@
     ADD_DEFINITIONS(-D_CRT_SECURE_NO_WARNINGS)
 ENDIF (WIN32)
 
-# Define the main library.
-ADD_LIBRARY(snappy SHARED
-        snappy-c.cc
+set(SNAPPY_SRCS snappy-c.cc
         snappy-c.h
         snappy-sinksource.cc
         snappy-sinksource.h
@@ -83,9 +81,16 @@
         snappy.cc
         snappy.h)
 
-TARGET_COMPILE_DEFINITIONS(snappy PRIVATE -DHAVE_CONFIG_H)
+# Define the main library.
+ADD_LIBRARY(snappy-static STATIC ${SNAPPY_SRCS})
+
+TARGET_COMPILE_DEFINITIONS(snappy-static PRIVATE -DHAVE_CONFIG_H)
 
-SET_TARGET_PROPERTIES(snappy PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS ON)
+if (MSVC)
+    SET_TARGET_PROPERTIES(snappy-static PROPERTIES OUTPUT_NAME snappy_static)
+else()
+    SET_TARGET_PROPERTIES(snappy-static PROPERTIES OUTPUT_NAME snappy)
+endif()
 
 INSTALL(FILES snappy.h
         snappy-c.h
@@ -93,15 +98,13 @@
         ${Snappy_BINARY_DIR}/snappy-stubs-public.h
         DESTINATION include)
 
-INSTALL(TARGETS snappy
+INSTALL(TARGETS snappy-static
         EXPORT SnappyTargets
         RUNTIME DESTINATION bin
         LIBRARY DESTINATION lib
         ARCHIVE DESTINATION lib)
 INSTALL(EXPORT SnappyTargets NAMESPACE Snappy:: DESTINATION lib/cmake/Snappy)
 
-SET_TARGET_PROPERTIES(snappy PROPERTIES VERSION ${PROJECT_VERSION}
-                      SOVERSION ${PROJECT_VERSION_MAJOR})
 
 SET(INCLUDE_INSTALL_DIR include)
 SET(LIBRARY_INSTALL_DIR lib)
@@ -127,15 +130,4 @@
 
 IF (HAVE_LIBLZO2)
     LIST(APPEND COMPRESSION_LIBS lzo2)
-ENDIF (HAVE_LIBLZO2)
-
-ADD_EXECUTABLE(snappy-unittest snappy_unittest.cc snappy-test.cc)
-TARGET_COMPILE_DEFINITIONS(snappy-unittest PRIVATE -DHAVE_CONFIG_H)
-TARGET_LINK_LIBRARIES(snappy-unittest snappy ${COMPRESSION_LIBS}
-                      ${GFLAGS_LIBRARIES})
-TARGET_INCLUDE_DIRECTORIES(snappy-unittest BEFORE PRIVATE ${Snappy_SOURCE_DIR}
-                           ${GTEST_INCLUDE_DIRS} ${GFLAGS_INCLUDE_DIRS})
-
-ADD_TEST(NAME snappy-unittest
-         WORKING_DIRECTORY ${Snappy_SOURCE_DIR}
-         COMMAND ${Snappy_BINARY_DIR}/snappy-unittest)
+ENDIF (HAVE_LIBLZO2)
\ No newline at end of file
