From: DeadSix27 <DeadSix27@users.noreply.github.com>
Date: Sun, 15 Apr 2017 07:42:49 +0200
Subject: [PATCH] fix static build on mingw

diff -Nur libzip_git.patched/lib/zip.h libzip_git/lib/zip.h
--- libzip_git/lib/zip.h	2017-04-15 07:41:49.422447521 +0200
+++ libzip_git.patched/lib/zip.h	2017-04-15 07:42:34.718870299 +0200
@@ -37,9 +37,9 @@
 
 #ifndef ZIP_EXTERN
 # ifndef ZIP_STATIC
-#  ifdef _WIN32
+#  ifdef _MSC_VER
 #   define ZIP_EXTERN __declspec(dllimport)
-#  elif defined(__GNUC__) && __GNUC__ >= 4
+#  elif (defined(__GNUC__) && __GNUC__ >= 4) && !defined(_WIN32)
 #   define ZIP_EXTERN __attribute__ ((visibility ("default")))
 #  else
 #   define ZIP_EXTERN
diff -Nur libzip_git.patched/libzip.pc.in libzip_git/libzip.pc.in
--- libzip_git.patched/libzip.pc.in	2017-04-15 07:41:49.406446916 +0200
+++ libzip_git/libzip.pc.in	2017-04-15 07:43:03.614673563 +0200
@@ -9,6 +9,6 @@
 Name: libzip
 Description: library for handling zip archives
 Version: @VERSION@
-Libs: @PKG_CONFIG_RPATH@ -L${libdir} -lzip @LIBS@
+Libs: @PKG_CONFIG_RPATH@ -L${libdir} -lzip -lbz2
 Cflags: -I${includedir} -I${libincludedir}
 
