From 55129161b3ababbfc0188baa2b034a01fc78fd64 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sun, 25 Dec 2016 12:55:51 +0100
Subject: [PATCH 2/2] -std=c++14 is required for GCC < 6

---
 gyp/common_defines.gypi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gyp/common_defines.gypi b/gyp/common_defines.gypi
index 72cc852eb..6c33b2e15 100644
--- a/gyp/common_defines.gypi
+++ b/gyp/common_defines.gypi
@@ -39,7 +39,7 @@
     ],
     'cflags_cc':
     [
-        '-std=c++11',
+        '-std=c++14',
         '-fpermissive',
         '-msse2',
         '-O2',
-- 
2.11.0

