From aace54299e8b1b484177788cf6808cb54caacd60 Mon Sep 17 00:00:00 2001
From: DeadSix27 <DeadSix27@users.noreply.github.com>
Date: Wed, 11 Apr 2018 22:35:54 +0200
Subject: [PATCH] disable shader optimization due to crashes

---
 video/out/gpu/spirv_shaderc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/video/out/gpu/spirv_shaderc.c b/video/out/gpu/spirv_shaderc.c
index ee702053d5..64f0dbcfda 100644
--- a/video/out/gpu/spirv_shaderc.c
+++ b/video/out/gpu/spirv_shaderc.c
@@ -32,7 +32,7 @@ static bool shaderc_init(struct ra_ctx *ctx)
         goto error;
 
     shaderc_compile_options_set_optimization_level(p->opts,
-                                            shaderc_optimization_level_size);
+                                            shaderc_optimization_level_zero);
     if (ctx->opts.debug)
         shaderc_compile_options_set_generate_debug_info(p->opts);
 
-- 
2.17.0

