diff --git a/Makefile.am b/Makefile.am
index 8c516fb..f350c56 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,12 +4,11 @@
 ACLOCAL_AMFLAGS = -I m4
 
 ## The order of subdirs is important, don't change without a reason.
-SUBDIRS = gen.tab lib bin doc test
+SUBDIRS = gen.tab lib bin
 
 EXTRA_DIST = autogen.sh ChangeLog.old \
-	meson.build meson_options.txt bin/meson.build doc/meson.build \
-	gen.tab/meson.build lib/meson.build test/meson.build \
-	test/test-runner.py test/unicode-conformance/meson.build \
+	meson.build meson_options.txt bin/meson.build \
+	gen.tab/meson.build lib/meson.build \
         README.md
 
 pkgconfigdir = $(libdir)/pkgconfig
