From ffb08aa48a2c741d1df02e43df0ab14b1e150334 Mon Sep 17 00:00:00 2001
From: DeadSix27 <DeadSix27@users.noreply.github.com>
Date: Sat, 7 Dec 2019 16:19:15 +0100
Subject: [PATCH] remove tests & examples

---
 CMakeLists.txt | 24 +-----------------------
 1 file changed, 1 insertion(+), 23 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0fe939d..9d18bf1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -224,26 +224,4 @@ if(NOT SKIP_INSTALL_FILES AND NOT SKIP_INSTALL_ALL )
 endif()
 if(NOT SKIP_INSTALL_FILES AND NOT SKIP_INSTALL_ALL )
     install(FILES ${ZLIB_PC} DESTINATION "${INSTALL_PKGCONFIG_DIR}")
-endif()
-
-#============================================================================
-# Example binaries
-#============================================================================
-
-add_executable(example test/example.c)
-target_link_libraries(example zlib)
-add_test(example example)
-
-add_executable(minigzip test/minigzip.c)
-target_link_libraries(minigzip zlib)
-
-if(HAVE_OFF64_T)
-    add_executable(example64 test/example.c)
-    target_link_libraries(example64 zlib)
-    set_target_properties(example64 PROPERTIES COMPILE_FLAGS "-D_FILE_OFFSET_BITS=64")
-    add_test(example64 example64)
-
-    add_executable(minigzip64 test/minigzip.c)
-    target_link_libraries(minigzip64 zlib)
-    set_target_properties(minigzip64 PROPERTIES COMPILE_FLAGS "-D_FILE_OFFSET_BITS=64")
-endif()
+endif()
\ No newline at end of file
-- 
2.24.0.windows.2

