diff -Nur fontconfig_git.orig/src/fccache.c fontconfig_git/src/fccache.c
--- fontconfig_git.orig/src/fccache.c	2019-04-01 01:31:59.246660540 +0200
+++ fontconfig_git/src/fccache.c	2019-04-01 01:32:40.949993873 +0200
@@ -46,6 +46,17 @@
 #define O_BINARY 0
 #endif
 
+#ifdef __MINGW32__
+#include <sys/utime.h>
+int utimes(const char *filename, const struct timeval times[2]) {
+	struct _utimbuf timebuffer;
+	timebuffer.actime = times[0].tv_sec;
+	timebuffer.modtime = times[1].tv_sec;
+
+	return _utime(filename, &timebuffer);
+}
+#endif
+
 FcBool
 FcDirCacheCreateUUID (FcChar8  *dir,
 		      FcBool    force,
