diff --git a/Project/GNU/Library/configure.ac b/Project/GNU/Library/configure.ac
index 9aa1dfe..4927388 100644
--- a/Project/GNU/Library/configure.ac
+++ b/Project/GNU/Library/configure.ac
@@ -359,8 +359,8 @@ if test -e ../../../../ZenLib/Project/GNU/Library/libzen-config; then
 elif test "$(command -v libzen-config)" ; then
 	enable_unicode="$(libzen-config Unicode)"
 else
-	if pkg-config --exists libzen; then
-		enable_unicode="$(pkg-config --variable=Unicode libzen)"
+	if $PKG_CONFIG --exists libzen; then
+		enable_unicode="$($PKG_CONFIG --variable=Unicode libzen)"
 	else
 		AC_MSG_ERROR([libzen configuration is not found])
 	fi
@@ -440,15 +440,15 @@ elif test "$(command -v libzen-config)" ; then
 		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(libzen-config LIBS)"
 	fi
 else
-	if pkg-config --exists libzen; then
-		CXXFLAGS="$CXXFLAGS $(pkg-config --cflags libzen)"
+	if $PKG_CONFIG --exists libzen; then
+		CXXFLAGS="$CXXFLAGS $($PKG_CONFIG --cflags libzen)"
 		if test "$enable_staticlibs" = "yes"; then
 			with_zenlib="system (static)"
-			LIBS="$LIBS $(pkg-config --variable=LIBS_Static libzen)"
+			LIBS="$LIBS $($PKG_CONFIG --variable=LIBS_Static libzen)"
 		else
 			with_zenlib="system"
-			LIBS="$LIBS $(pkg-config --libs libzen)"
-			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --variable=LIBS libzen)"
+			LIBS="$LIBS $($PKG_CONFIG --libs libzen)"
+			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $($PKG_CONFIG --variable=LIBS libzen)"
 		fi
 	else
 		AC_MSG_ERROR([libzen configuration is not found])
@@ -468,15 +468,15 @@ if test -d $with_libcurl; then
 			AC_MSG_ERROR([Problem while configuring builtin curl (libcurl.pc not found)])
 		fi
 	fi
-	CXXFLAGS="$CXXFLAGS $(pkg-config --cflags $libcurlpcfile)"
+	CXXFLAGS="$CXXFLAGS $($PKG_CONFIG --cflags $libcurlpcfile)"
 	if test "$enable_staticlibs" = "yes"; then
 		using_libcurl="custom (static)"
-		LIBS="$LIBS $(pkg-config --libs --static $libcurlpcfile)"
-		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --libs --static $libcurlpcfile)"
+		LIBS="$LIBS $($PKG_CONFIG --libs --static $libcurlpcfile)"
+		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $($PKG_CONFIG --libs --static $libcurlpcfile)"
 	else
 		using_libcurl="custom"
-		LIBS="$LIBS $with_libcurl/lib $(pkg-config --libs $libcurlpcfile)"
-		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(pkg-config --libs $libcurlpcfile)"
+		LIBS="$LIBS $with_libcurl/lib $($PKG_CONFIG --libs $libcurlpcfile)"
+		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $($PKG_CONFIG --libs $libcurlpcfile)"
 	fi
 elif test "$with_libcurl" = "no"; then
 	CXXFLAGS="$CXXFLAGS -DMEDIAINFO_LIBCURL_NO"
@@ -488,23 +488,23 @@ elif test -e ../../../../curl/libcurl.pc; then
 	CXXFLAGS="$CXXFLAGS -I../../../../curl/include"
 	if test "$enable_staticlibs" = "yes"; then
 		using_libcurl="builtin (static)"
-		LIBS="$LIBS $(pkg-config --libs --static ../../../../curl/libcurl.pc)"
+		LIBS="$LIBS $($PKG_CONFIG --libs --static ../../../../curl/libcurl.pc)"
 	else
 		using_libcurl="builtin"
-		LIBS="$LIBS $(pkg-config --libs ../../../../curl/libcurl.pc)"
-		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(pkg-config --libs ../../../../curl/libcurl.pc)"
-		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --libs ../../../../curl/libcurl.pc)"
+		LIBS="$LIBS $($PKG_CONFIG --libs ../../../../curl/libcurl.pc)"
+		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $($PKG_CONFIG --libs ../../../../curl/libcurl.pc)"
+		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $($PKG_CONFIG --libs ../../../../curl/libcurl.pc)"
 	fi
-elif pkg-config --exists libcurl; then
-	CXXFLAGS="$CXXFLAGS $(pkg-config --cflags libcurl)"
+elif $PKG_CONFIG --exists libcurl; then
+	CXXFLAGS="$CXXFLAGS $($PKG_CONFIG --cflags libcurl)"
 	if test "$enable_staticlibs" = "yes"; then
 		using_libcurl="system (static)"
-		LIBS="$LIBS $(pkg-config --libs --static libcurl)"
+		LIBS="$LIBS $($PKG_CONFIG --libs --static libcurl)"
 	else
 		using_libcurl="system"
-		LIBS="$LIBS $(pkg-config --libs libcurl)"
-		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(pkg-config --libs libcurl)"
-		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --libs libcurl)"
+		LIBS="$LIBS $($PKG_CONFIG --libs libcurl)"
+		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $($PKG_CONFIG --libs libcurl)"
+		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $($PKG_CONFIG --libs libcurl)"
 	fi
 elif test -e /usr/bin/curl-config; then
 	CXXFLAGS="$CXXFLAGS $(/usr/bin/curl-config --cflags)"
@@ -513,7 +513,7 @@ elif test -e /usr/bin/curl-config; then
 		LIBS="$LIBS $(/usr/bin/curl-config --libs)"
 	else
 		using_libcurl="system"
-		LIBS="$LIBS $(pkg-config --libs libcurl)"
+		LIBS="$LIBS $($PKG_CONFIG --libs libcurl)"
 		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(/usr/bin/curl-config --libs)"
 		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(/usr/bin/curl-config --libs)"
 	fi
@@ -526,14 +526,14 @@ elif test -e ../../../../Shared/Project/curl/Compile.sh; then
 		CXXFLAGS="$CXXFLAGS -I../../../../Shared/Source/curl/include"
 		if test "$enable_staticlibs" = "yes"; then
 			using_libcurl="builtin (static)"
-			LIBS="$LIBS -L../../../../Shared/Source/curl/lib $(pkg-config --libs --static ../../../../Shared/Source/curl/libcurl.pc)"
-			MediaInfoLib_LIBS="$MediaInfoLib_LIBS -L../../../../Shared/Source/curl/lib $(pkg-config --libs --static ../../../../Shared/Source/curl/libcurl.pc)"
-			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static -L../../../../Shared/Source/curl/lib $(pkg-config --libs --static ../../../../Shared/Source/curl/libcurl.pc)"
+			LIBS="$LIBS -L../../../../Shared/Source/curl/lib $($PKG_CONFIG --libs --static ../../../../Shared/Source/curl/libcurl.pc)"
+			MediaInfoLib_LIBS="$MediaInfoLib_LIBS -L../../../../Shared/Source/curl/lib $($PKG_CONFIG --libs --static ../../../../Shared/Source/curl/libcurl.pc)"
+			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static -L../../../../Shared/Source/curl/lib $($PKG_CONFIG --libs --static ../../../../Shared/Source/curl/libcurl.pc)"
 		else
 			using_libcurl="builtin"
-			LIBS="$LIBS -L../../../../Shared/Source/curl/lib $(pkg-config --libs ../../../../Shared/Source/curl/libcurl.pc)"
-			MediaInfoLib_LIBS="$MediaInfoLib_LIBS -L../../../../Shared/Source/curl/lib $(pkg-config --libs ../../../../Shared/Source/curl/libcurl.pc)"
-			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static -L../../../../Shared/Source/curl/lib $(pkg-config --libs ../../../../Shared/Source/curl/libcurl.pc)"
+			LIBS="$LIBS -L../../../../Shared/Source/curl/lib $($PKG_CONFIG --libs ../../../../Shared/Source/curl/libcurl.pc)"
+			MediaInfoLib_LIBS="$MediaInfoLib_LIBS -L../../../../Shared/Source/curl/lib $($PKG_CONFIG --libs ../../../../Shared/Source/curl/libcurl.pc)"
+			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static -L../../../../Shared/Source/curl/lib $($PKG_CONFIG --libs ../../../../Shared/Source/curl/libcurl.pc)"
 		fi
 	else
 		AC_MSG_ERROR([Problem while compiling builtin curl])
@@ -546,43 +546,43 @@ dnl -------------------------------------------------------------------------
 dnl libmms
 dnl
 if test -d $with_libmms; then
-	CXXFLAGS="$CXXFLAGS -DMEDIAINFO_LIBMMS_FROMSOURCE -I$with_libmms/src $(pkg-config --cflags $with_libmms/pkgconfig/libmms.pc)"
-	MediaInfoLib_CXXFLAGS="$MediaInfoLib_CXXFLAGS -I$with_libmms $(pkg-config --cflags $with_libmms/pkgconfig/libmms.pc)"
+	CXXFLAGS="$CXXFLAGS -DMEDIAINFO_LIBMMS_FROMSOURCE -I$with_libmms/src $($PKG_CONFIG --cflags $with_libmms/pkgconfig/libmms.pc)"
+	MediaInfoLib_CXXFLAGS="$MediaInfoLib_CXXFLAGS -I$with_libmms $($PKG_CONFIG --cflags $with_libmms/pkgconfig/libmms.pc)"
 	if test "$enable_staticlibs" = "yes"; then
 		using_libmms="custom (static)"
-		LIBS="$LIBS -L$with_libmms/src/.libs $(pkg-config --libs --static $with_libmms/pkgconfig/libmms.pc)"
-		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static -L$with_libmms/src/.libs $(pkg-config --libs --static $with_libmms/pkgconfig/libmms.pc)"
+		LIBS="$LIBS -L$with_libmms/src/.libs $($PKG_CONFIG --libs --static $with_libmms/pkgconfig/libmms.pc)"
+		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static -L$with_libmms/src/.libs $($PKG_CONFIG --libs --static $with_libmms/pkgconfig/libmms.pc)"
 	else
 		using_libmms="custom"
-		LIBS="$LIBS -L$with_libmms/src/.libs $(pkg-config --libs $with_libmms/pkgconfig/libmms.pc)"
-		MediaInfoLib_LIBS="$MediaInfoLib_LIBS -L$with_libmms/src/.libs $(pkg-config --libs $with_libmms/pkgconfig/libmms.pc)"
+		LIBS="$LIBS -L$with_libmms/src/.libs $($PKG_CONFIG --libs $with_libmms/pkgconfig/libmms.pc)"
+		MediaInfoLib_LIBS="$MediaInfoLib_LIBS -L$with_libmms/src/.libs $($PKG_CONFIG --libs $with_libmms/pkgconfig/libmms.pc)"
 	fi
 elif test "$with_libmms" = "no"; then
 	CXXFLAGS="$CXXFLAGS -DMEDIAINFO_LIBMMS_NO"
 	using_libmms="no"
 elif test -e ../../../../libmms/libmms.pc; then
-	CXXFLAGS="$CXXFLAGS $(pkg-config --cflags ../../../../libmms/libmms.pc)"
-	MediaInfoLib_CXXFLAGS="$MediaInfoLib_CXXFLAGS $(pkg-config --cflags ../../../../libmms/libmms.pc)"
+	CXXFLAGS="$CXXFLAGS $($PKG_CONFIG --cflags ../../../../libmms/libmms.pc)"
+	MediaInfoLib_CXXFLAGS="$MediaInfoLib_CXXFLAGS $($PKG_CONFIG --cflags ../../../../libmms/libmms.pc)"
 	if test "$enable_staticlibs" = "yes"; then
 		using_libmms="builtin (static)"
-		LIBS="$LIBS $(pkg-config --libs --static ../../../../libmms/libmms.pc)"
+		LIBS="$LIBS $($PKG_CONFIG --libs --static ../../../../libmms/libmms.pc)"
 	else
 		using_libmms="builtin"
-		LIBS="$LIBS $(pkg-config --libs ../../../../libmms/libmms.pc)"
-		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(pkg-config --libs ../../../../libmms/libmms.pc)"
-		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --libs ../../../../libmms/libmms.pc)"
+		LIBS="$LIBS $($PKG_CONFIG --libs ../../../../libmms/libmms.pc)"
+		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $($PKG_CONFIG --libs ../../../../libmms/libmms.pc)"
+		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $($PKG_CONFIG --libs ../../../../libmms/libmms.pc)"
 	fi
-elif pkg-config --exists libmms; then
-	CXXFLAGS="$CXXFLAGS $(pkg-config --cflags libmms)"
-	MediaInfoLib_CXXFLAGS="$MediaInfoLib_CXXFLAGS $(pkg-config --cflags libmms)"
+elif $PKG_CONFIG --exists libmms; then
+	CXXFLAGS="$CXXFLAGS $($PKG_CONFIG --cflags libmms)"
+	MediaInfoLib_CXXFLAGS="$MediaInfoLib_CXXFLAGS $($PKG_CONFIG --cflags libmms)"
 	if test "$enable_staticlibs" = "yes"; then
 		using_libmms="system (static)"
-		LIBS="$LIBS $(pkg-config --libs --static libmms)"
+		LIBS="$LIBS $($PKG_CONFIG --libs --static libmms)"
 	else
 		using_libmms="system"
-		LIBS="$LIBS $(pkg-config --libs libmms)"
-		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(pkg-config --libs libmms)"
-		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --libs libmms)"
+		LIBS="$LIBS $($PKG_CONFIG --libs libmms)"
+		MediaInfoLib_LIBS="$MediaInfoLib_LIBS $($PKG_CONFIG --libs libmms)"
+		MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $($PKG_CONFIG --libs libmms)"
 	fi
 else
 	CXXFLAGS="$CXXFLAGS -DMEDIAINFO_LIBMMS_NO"