--- a/Project/GNU/Library/configure.ac
+++ b/Project/GNU/Library/configure.ac
@@ -172,21 +172,6 @@
 dnl
 
 dnl -------------------------------------------------------------------------
-dnl Check if size_t is long
-dnl
-AC_LANG([C++])
-AC_COMPILE_IFELSE([AC_LANG_SOURCE([
-	#include <cstdlib>
-	void foo(signed int) {}
-	void foo(unsigned int) {}
-	int main () {foo(size_t(0)); return 0;}])], [size_t_is_long="no"], [size_t_is_long="yes"])
-
-if test "$size_t_is_long" = "yes"; then
-	CXXFLAGS="$CXXFLAGS -DSIZE_T_IS_LONG"
-	ZenLib_CXXFLAGS="$ZenLib_CXXFLAGS -DSIZE_T_IS_LONG"
-fi
-
-dnl -------------------------------------------------------------------------
 dnl Check if stream is available
 dnl
 AC_LANG([C++])
