diff -Nur pcre2-10.23.original/src/pcre2grep.c pcre2-10.23/src/pcre2grep.c
--- pcre2-10.23.original/src/pcre2grep.c        2017-07-27 17:29:26.687957476 +0200
+++ pcre2-10.23/src/pcre2grep.c 2017-07-27 17:30:04.845030039 +0200
@@ -628,7 +628,7 @@

 /************* Directory scanning Unix-style and z/OS ***********/

-#if (defined HAVE_SYS_STAT_H && defined HAVE_DIRENT_H && defined HAVE_SYS_TYPES_H) || defined NATIVE_ZOS
+#if ((defined HAVE_SYS_STAT_H && defined HAVE_DIRENT_H && defined HAVE_SYS_TYPES_H) || defined NATIVE_ZOS) && !defined WIN32
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <dirent.h>